MetriCam2/Camera.cs

```diff
@@ -683,7 +683,10 @@ public bool IsValid(object value)
                     { /* empty */ }
                     catch (OverflowException)
                     { /* empty */ }
+
+                    isTypeConvertible = (null != castedValue);
```

Comment:
Is das korrekt, diese Zuweisung aus dem try-Block rauszuziehen?
Nehmen wir an, bei `ChangeType` wuerde eine Exception fliegen. Dann wuerde ich erwarten, dass `isTypeConvertible` den Wert `false` bekommt.
`castedValue` hat dann allerdings den Wert `default(T)`, und das ist bei einem ValueType nicht `null`, also waere `isTypeConvertible` dann `true`.